tshirtsn_test.cpp

```diff
@@ -0,0 +1,14 @@
+#include <assert.h>
+#include <iostream>
+#include <vector>
+#include <string>
+#include "./tshirtsn.h"
+
+int main() {
+    assert(size_measure(38) == 'S');

```

Comment:
What's the reason to chose 38 to be `S` and not `M`? How would this relate to the customer experience?